app/controllers/orga/submissions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Orga::SubmissionsController < Orga::BaseController
+  before_action :find_mailing
+  before_action :set_submission, only: :update
```

Comment:
Again, if this is only used for **one specific** action, I'd suggest to instead make it an explicit call within the specific action. Especially given the fact that there is already another `before_action` for the respective action and they now depend on their order...